_redirects

```diff
@@ -56,6 +56,7 @@ https://babel.netlify.com/* https://babeljs.io/:splat 301!
 /docs/usage/options/                /docs/options
 /caveats.html                       /docs/caveats
 /repl.html                          /repl/
+/en/repl                            /repl/
```

Comment:
```suggestion
/repl.html                          /repl
/en/repl                            /repl
```

actually `/repl/` is not working: https://deploy-preview-2742--babel.netlify.app/repl/

Reply:
aren't trailing slash supposed to work as well? Im confused?

Reply:
Well the external scripts are sourced as `"./repl/repl.js"`, they won't be loaded if there is trailing slash because it would be `repl/repl/repl.js`.